nss-evening-cohort-11/WildflowerCoffeeGifts
# Ars control add to cart button

This PR adds toggling between buttons in a couple of places depending one whether user is logged in or not.

To test:
* To check Log In / Log Out toggle:
1. Log out of the app. 
1. Go to the Login page. ***Expected result***: You should see only the Log In button and you should also see the registration fields. 
1. Log in. 
1. Go back to the Login page.  ***Expected result***: You should see only the Log OUT button and you should NOT see the registration fields. 

* To check Add to cart / Please log in buttons:
1. Log out of the app. 
1. Go to the Single Product View page for any product. ***Expected result***: You should see the Please log in .... button. 
1. Log in. 
1. Go back to the Single Product View page.  ***Expected result***: You should see the Add to Cart button.
